src/azure-cli/azure/cli/command_modules/acr/_docker_utils.py

```diff
@@ -655,6 +657,32 @@ def request_data_from_registry(http_method,
     raise CLIError(errorMessage)
 
 
+def parse_image_name(image, allow_digest=False, default_latest=True):
```

Comment:
Is the parse function new? Do you need to add a unit test for the parsing logic?

Reply:
No I just moved it to the docker utils file since it's being used by multiple commands. It didn't make sense to keep it under registry. But the method is already being tested through several commands already. I don't think there's any need to add a test just for it.